appuio/redis/templates/recreate.yaml

```
@@ -0,0 +1,76 @@
+{{- if .Values.forceUpdate }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sts-deleter
```

```suggestion
  name: {{ printf "%s-sts-deleter" (include "common.names.fullname" .) }}
```

Always use the fullname template or setting fullNameOverride/* can lead to unexpected results.

If no additional pre/suffix is required:
```yaml
name: {{ template "common.names.fullname" . }}
```